Assets/MixedRealityToolkit.Extensions/Providers/Experimental/UnityAR/UnityARCameraSettings.cs

```diff
@@ -37,38 +37,36 @@ public class UnityARCameraSettings : BaseDataProvider, IMixedRealityCameraSettin
             string name = null,
             uint priority = DefaultPriority,
             BaseCameraSettingsProfile profile = null) : base(cameraSystem, name, priority, profile)
-        { }
-
-#region IMixedRealityCameraSettings
+        {
+            ReadProfile();
+        }
 
-        /// <inheritdoc/>
-        public bool IsOpaque => false;
+        private ArTrackedPose poseSource = ArTrackedPose.ColorCamera;
+        private ArTrackingType trackingType = ArTrackingType.RotationAndPosition;
+        private ArUpdateType updateType = ArUpdateType.UpdateAndBeforeRender;
 
-        /// <inheritdoc/>
-        public void ApplyDisplaySettings()
+        private void ReadProfile()
         {
-            MixedRealityCameraProfile cameraProfile = (Service as IMixedRealityCameraSystem)?.CameraProfile;
-            if (cameraProfile == null) { return; } 
-
-            if (IsOpaque)
+            if (SettingsProfile == null)
             {
-                CameraCache.Main.clearFlags = cameraProfile.CameraClearFlagsOpaqueDisplay;
-                CameraCache.Main.nearClipPlane = cameraProfile.NearClipPlaneOpaqueDisplay;
-                CameraCache.Main.farClipPlane = cameraProfile.FarClipPlaneOpaqueDisplay;
-                CameraCache.Main.backgroundColor = cameraProfile.BackgroundColorOpaqueDisplay;
-                QualitySettings.SetQualityLevel(cameraProfile.OpaqueQualityLevel, false);
-            }
-            else
-            {
-                CameraCache.Main.clearFlags = cameraProfile.CameraClearFlagsTransparentDisplay;
-                CameraCache.Main.backgroundColor = cameraProfile.BackgroundColorTransparentDisplay;
-                CameraCache.Main.nearClipPlane = cameraProfile.NearClipPlaneTransparentDisplay;
-                CameraCache.Main.farClipPlane = cameraProfile.FarClipPlaneTransparentDisplay;
-                QualitySettings.SetQualityLevel(cameraProfile.TransparentQualityLevel, false);
+                Debug.LogWarning("A profile was not specified for the XR Camera Settings provider.\nApplying Microsoft Mixed Reality Toolkit default options.");
```

Comment:
> Applying Microsoft Mixed Reality Toolkit default options.

Where does this happen?

Reply:
in ReadProfile... doesn't it?

Reply:
Oh. I see.... It's in the declarations of the variables. Is that a confusing pattern? I can change it.